packages/twenty-server/src/engine/workspace-cache-storage/workspace-cache-storage.service.ts

```diff
@@ -55,7 +55,7 @@ export class WorkspaceCacheStorageService {
     return this.cacheStorageService.set<EntitySchemaOptions<any>[]>(
       `${WorkspaceCacheKeys.ORMEntitySchemas}:${workspaceId}:${metadataVersion}`,
       entitySchemas,
-      TTL_INFINITE,
+      TTL_ONE_WEEK,
```

Comment:
Additional change as the cache is now able to auto-recover quickly if some cache keys are missing